catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceMetacardTypeImpl.java

```diff
@@ -61,6 +63,13 @@
                 true    /* multivalued */,
                 BasicTypes.XML_TYPE));
 
+        DESCRIPTORS.add(new AttributeDescriptorImpl(WORKSPACE_IS_ACTIVE,
```

Comment:
Why do workspaces still need an active flag?  If a local workspace exists and its ID is in the subscription persistence store, then it is active.